persistence/pg.go

```diff
@@ -131,6 +148,23 @@ func (p *PostgresPersister) Get(ctx context.Context, hash lntypes.Hash) (*Invoic
 	return invoice, htlcs, nil
 }
 
+func (p *PostgresPersister) GetSettledInvoices(ctx context.Context, numMaxInvoices, invoiceOffset int) ([]*dbInvoice, error) {
+	var dbInvoices []*dbInvoice
+	err := p.conn.RunInTransaction(ctx, func(tx *pg.Tx) error {
+		return tx.ModelContext(ctx, &dbInvoices).
+			Where("sequence_num>?", invoiceOffset).
```

Comment:
`invoiceOffset` is different again from index and sequence. Maybe `sequenceStart`?